### ISSUE TITLE
Train a model from scratch vs fine-tuning 

### ISSUE THREAD

**Issue by tjasmin111** [ultralytics/ultralytics#16053]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I'm training a 2 class detection model with Yolov8 Nano. I've iterated 5 times and finetuned the model. Recently I added more data and the size is now 2x (200000 images now). Should I fine tune the previous model or better to train from scratch?

And how do I modify it to train from scratch?

`yolo detect train ... data= data.yaml`

**Comment by Y-T-G** [comment]
It's usually better to train from the pretrained checkpoints.

**Comment by tjasmin111** [comment]
Got it. So you mean I use model=yolov8n.pt and not pretrained=best.pt (my previous model). Is that right?

And what's the rationale behind it?

**Comment by Y-T-G** [comment]
Yes. Neural networks experience problems when warm-starting.

https://pureai.com/Articles/2021/02/01/warm-start-ml.aspx

**Comment by tjasmin111** [comment]
Interesting article. Can we apply a shrink-perturb training to yolov8?
Btw, another things is that my pretrained model was a prinwd model. Should I train with a scratch model, prune, and then fine tune again?

**Comment by tjasmin111** [comment]
> Got it. So you mean I use model=yolov8n.pt and not pretrained=best.pt (my previous model). Is that right?
> 
> And what's the rationale behind it?

So I was trying to train the model from scratch using `model=yolov8n.pt`. I realized there is an flag called `pretrained` too. Should I  use it and set it to False?

It's `True` by default which it says: _Enhances training efficiency and model performance._

**Comment by glenn-jocher** [comment]
Yes, use `model=yolov8n.pt` to start from scratch. The `pretrained` flag is for using weights from a previous run; setting it to `False` is fine if you're starting fresh.
